learn/polkadot/parachains.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Parachains and parathreads
+description: Introduction to the basic concepts of a parachain and parathreads
+---
+
+# Parachains and Parathreads in Polkadot
+
+## Parachains
```

Comment:
Augusto started using header attributes on his PR, so I would recommend being consistent and including them on all the pages.

Reply:
By header attributes, I mean `{: #parachains }`